systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziDowngradeIsolatedST.java

```diff
@@ -56,7 +56,7 @@ private void performDowngrade(JsonObject testParameters, ExtensionContext extens
         // Setup env
         // We support downgrade only when you didn't upgrade to new inter.broker.protocol.version and log.message.format.version
         // https://strimzi.io/docs/operators/latest/full/deploying.html#con-target-downgrade-version-str
-        setupEnvAndUpgradeClusterOperator(extensionContext, testParameters, producerName, consumerName, continuousTopicName, continuousConsumerGroup, testParameters.getString("deployKafkaVersion"), clusterOperator.getDeploymentNamespace());
+        setupEnvAndUpgradeClusterOperator(extensionContext, testParameters, producerName, consumerName, continuousTopicName, continuousConsumerGroup, testParameters.getDeployKafkaVersion(), clusterOperator.getDeploymentNamespace());
```

Comment:
I guess you can pass there just `testParameters` and remove the `testParameters.getDeployKafkaVersion()`

Reply:
this is necessary for test when null is passed as unknown kafka version